### PR TITLE
refactor: Split test to upload files just once

### PR DESCRIPTION
**Pull request by CozyKourai**
Move & and add Create Folder, and upload file to `init-data`
Add `clean-data` to delete the folder

There are new names for screenshots (Init/Clean Data, PublicViewerFeature)
and some screenshots for ClassicationScenario will need to be validate (as there is a new folder in drive which shows up on screenshots)

### CODE REVIEW

File: testcafe/runner-drive.js [hunk 1]
```diff
@@ -17,14 +17,17 @@ async function runRunner() {
...
       //Tests !
       'testcafe/tests/drive/classification_scenario.js',
       'testcafe/tests/drive/navigation.js',
       'testcafe/tests/drive/folder_sharing_scenario.js',
       'testcafe/tests/drive/file_sharing_scenario.js',
       'testcafe/tests/drive/viewer-feature.js',
-      'testcafe/tests/drive/public-viewer-feature.js'
+      'testcafe/tests/drive/public-viewer-feature.js',
+      //cleanup
+      //Init data : Unzip archive with files to upload
```
Comment (y-lohse) [inline testcafe/runner-drive.js]: I'm confused by this comment, what does it mean?

File: testcafe/tests/drive/clean-data.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { FOLDER_NAME } from '../helpers/data'
+import { checkToastAppearsAndDisappears } from '../pages/commons'
+import { driveUser } from '../helpers/roles'
+//import { Role } from 'testcafe'
```
Comment (y-lohse) [inline testcafe/tests/drive/clean-data.js]: If we don't need this line, we might as well remove it :)

File: testcafe/tests/drive/clean-data.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { TESTCAFE_DRIVE_URL, SLUG } from '../helpers/utils'
+import { initVR } from '../helpers/visualreview-utils'
+let data = require('../helpers/data')
```
Comment (y-lohse) [inline testcafe/tests/drive/clean-data.js]: Why are we using a `let … require` instead of an `import` here?

File: testcafe/runner-drive.js (hunk 1 above)
Comment (CozyKourai) [reply testcafe/runner-drive.js]: It means I copy/paste too quickly! :)

File: testcafe/tests/drive/clean-data.js (hunk 3 above)
Comment (CozyKourai) [reply testcafe/tests/drive/clean-data.js]: change back to import

**Comment by CozyKourai** [comment]
[Upload drive data in a separate test](https://trello.com/c/v5orHV4j/1977-upload-drive-data-in-a-separate-test)

**Comment by CozyKourai** [comment]
[Upload drive data in a separate test](https://trello.com/c/v5orHV4j/1977-upload-drive-data-in-a-separate-test)

**Pull request merged**